bindgen/ir/location/LocationManager.cpp

```diff
@@ -0,0 +1,17 @@
+#include "LocationManager.h"
+#include "SourceLocation.h"
+
+LocationManager::LocationManager(std::string mainHeaderPath)
+    : mainHeaderPath(std::move(mainHeaderPath)) {}
+
+void LocationManager::loadConfig(const std::string &path) {
+    // TODO: implement
```

Comment:
Will this use the `stdHeader.config` file?

Reply:
Yes